scalafmt-tests/src/test/resources/scala3/OptionalBraces.stat

```diff
@@ -0,0 +1,130 @@
+#
+<<< simple value equals
+val test = // comm
+    val a = ""
+     a  +    ""
+>>>
+val test = // comm
+   val a = ""
+   a + ""
+<<< if else
+trait A /* comm */ :
+  val cond = if true then
+    stat1
+    stat2
+  else { // c1
+     stat3
+     stat4
+     }
+  end if
+>>>
+trait A /* comm */:
+   val cond =
+     if true then
+        stat1
+        stat2
+     else { // c1
+       stat3
+       stat4
+     }
+   end if
```

Comment:
wrong indentation here

Reply:
I haven't dug into that too much. I will try to figure exactly what is the problem here.

Reply:
i will look, too, if you said these two (`if` and `end if`) should be packaged as a block.

Reply:
Actually, this doesn't parse in Dotty:
```
  trait A /* comm */ :
    val cond = if true then
      stat1
      stat2
    else  // c1
      stat3
      stat4
    end if
```

It complains about misaligned end statement. You need an actual indentation to have an end statement (I think?)

Reply:
thank you, modified this!